scripts/utils/release.md

```diff
@@ -0,0 +1,33 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
It doesn't look like utility honestly ;)

Agreed to create `scripts/release` folder and move all release related files there.